webplugin/js/app/storage/cookie-service.js

```diff
@@ -0,0 +1,99 @@
+class KmCookieStorage extends KmStorage {
+    constructor() {
+        super();
+    }
+
+    /* There is no need to decode the cookie because we are not encoding the cookie */
```

Comment:
is it a new behaviour or earlier also we were not encoding? 
after this release, will there be any impact on the following case:

1. In the existing production release, User uses chat widget to send messages and some cookies are stored 
2. Next day, Kommunicate deploys this new version with the new implementation
3. Same user comes back and refresh the page and starting chatting.

Will it work seamlessly for the end user with the same cookies, same user id, chat history, etc?

Reply:
We have never encoded the cookie.